docs/warning-codes.md

```diff
@@ -92,4 +92,6 @@
 ### 45
 `An internal agent process failed to execute.`
 ### 46
-`A reserved eventType was provided to recordCustomEvent(...) -- The event was not recorded.`
\ No newline at end of file
+`A reserved eventType was provided to recordCustomEvent(...) -- The event was not recorded.`
+### 47
+`With fix_stylesheet off, ensure that all of your CSS are decorated with the "crossorigin='anonymous'" attribute or are publicly accessible. Replays may display bad layout otherwise.`
```

Comment:
Only warn if "fix_stylesheets" was on but we failed to fix it
```suggestion
`We tried to access a stylesheet's contents but failed due to browser security. For best results, ensure that cross-domain CSS assets are decorated with "crossorigin='anonymous'" attribution or are otherwise publicly accessible.`
```